dotnet/src/AutoGen.Anthropic/DTO/Tool.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Tool.cs
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace AutoGen.Anthropic.DTO;
+
+public class Tool
+{
+    [JsonPropertyName("name")]
+    public string? Name { get; set; }
+
+    [JsonPropertyName("description")]
+    public string? Description { get; set; }
+
+    [JsonPropertyName("input_schema")]
+    public InputSchema? InputSchema { get; set; }
+}
+
+public class InputSchema
+{
+    [JsonPropertyName("type")]
+    public string? _Type { get; set; }
```

Comment:
why not using `Type` as the property name here

Reply:
Updated.

I was too use to naming it `_Type` because of NAOT deserialization. When the fields are named after a reserved keyword, the generated files will cause an error. but for this case, i did not need to prefix it with an underscore. :)